tutorial/10_key_features/003_efficient_optimization_algorithms.py

```diff
@@ -72,7 +75,8 @@
 # though basically it is outperformed by :class:`optuna.pruners.SuccessiveHalvingPruner` and
 # :class:`optuna.pruners.HyperbandPruner` as in `this benchmark result <https://github.com/optuna/optuna/wiki/Benchmarks-with-Kurobako>`_.
 #
-#
+#.. seealso::
```

Comment:
I have little confidence in these following lines. Is it ok `#` exists in front of the new comments..?